pkg/kotsadm/copy_images.go

```diff
@@ -20,7 +20,7 @@ import (
 )
 
 // Copies Admin Console images from public registry to private registry
-func CopyImages(sourceEndpoint string, options types.PushImagesOptions, kotsNamespace string) error {
+func CopyImages(options types.PushImagesOptions, kotsNamespace string) error {
```

Comment:
This was only ever called with a `sourceEndpoint` of "docker.io" and even if it was different than "docker.io", it wasn't used as the image source, so I removed it.